docs/SVG_MAP_EDITOR.md

```diff
@@ -0,0 +1,70 @@
+# robot-sf
+
+## SVG Map Editor
+
+### About
+
+This file explains how to build a map for robot-sf using a svg-editor.
+All examples are made with [inkscape](https://inkscape.org/).
+![example](./img/inkscape_example.png)
+
+### Setup
+
+These settings should be checked before building a map.
+
+- Use **px** as global unit (File -> Document Properties)
+
+- Use absolute coordinates for your path, marked by the **M**.\
+(Edit -> Preferences -> Input/Output -> SVG Output -> Path string format -> Absolute)
+
+*Inkscape version: 1.3.2*
+
+[Further reference](https://github.com/ll7/robot_sf_ll7/issues/40)
+
+### Building the map
+
+Colours can be selected as desired, as the simulation uses its own colour scheme.
```

Comment:
Do you think it would be helpful to reference your colour scheme used in the example svg explicitly as rgb values?

Reply:
Yes it might be helpful to have a starting point. They colours can now be found in the colour section. 

I added the hex values because they are the ones I used to copy in Inkscape.